raster/r.statistics/o_var.c

```diff
@@ -17,7 +17,7 @@ int o_var(const char *basemap, const char *covermap, const char *outputmap,
 {
     struct Popen stats_child, reclass_child;
     FILE *stats, *reclass;
-    int first, i, count;
+    int first, i, count = 0;
```

Comment:
Wouldn't the others be uninitialized too?

Reply:
Others are not uninitialized.

And sorry about separate PRs I will group them from next time for similar issues.

Reply:
Ok, I see way down by uncollapsing more code that the first encounter of `first`, it assigns 1, and `i` is either initialized in the `if (first)` or with the `for (i = 0, i < value, i++) {` line.

Reply:
> And sorry about separate PRs I will group them from next time for similar issues.

No problem! It's really to help you and us get more done easily ;)

Reply:
> Ok, I see way down by uncollapsing more code that the first encounter of `first`, it assigns 1, and `i` is either initialized in the `if (first)` or with the `for (i = 0, i < value, i++) {` line.

Can someone jump in and confirm that I interpreted right?

Reply:
Of course your interpretation is correct.